### DISCUSSION TITLE
Getting weights of model while training

### DISCUSSION THREAD

**Discussion by adwaykanhere** [Project-MONAI/MONAI#5244] [Q&A] [answered]
I'm following the "3d_spleen_segmentation.ipynb" tutorial. 

While training I would like to extract and store the weights of the model. 
I don't see an attribute that allows be to get the weights of the model. 

Please suggest how I can do this?

**Comment by ebrahimebrahim** [comment, marked as answer]
You can follow the same suggestions that are given for PyTorch models in general [here](https://pytorch.org/tutorials/beginner/saving_loading_models.html).
